9.11/Program.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace _9._1
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите число меньше 500 ");
+            double a = double.Parse(Console.ReadLine());
+            Console.ReadKey();
+            while (a < 500)
+            {
+                double cube = Math.Pow(a, 3);
+                Console.WriteLine($"куб числа: {a} равен: {cube} ");
+                a++;
+            }
+
+
+
```

Comment:
1. Оператор `Console.ReadKey();` следует поставить в конец метода.
2. Старайтесь отделять пустой строкой смысловые части кода, например, циклы и операторы условия.
3. С другой стороны, не злоупотребляйте пустыми строками.
4. Маленькие степени (квадрат, куб) лучше вычислять через умножение.
5. Пишите код проще.
```suggestion            
            while (a < 500)
            {
                Console.WriteLine($"куб числа {a} равен {a * a * a}");
                a++;
            }

            Console.ReadKey();
```